backend/src/main/java/com/sidedish/domain/EventCategory.java

```diff
@@ -0,0 +1,15 @@
+package com.sidedish.domain;
+
+public enum EventCategory {
+	MEAT("풍성한 고기 반찬"), SIMPLE("편리한 반찬 세트"), SEASON("맛있는 제철 요리"), KIDS("우리 아이 영양 반찬");
+
+	private final String name;
+
+	EventCategory(String name) {
```

Comment:
여기도 마찬가지로 `final` 이면 좋겠네요. 왜 일까요?
```suggestion
	EventCategory(final String name) {
```